crud/select.lua

```diff
@@ -229,11 +229,16 @@ function select_module.pairs(space_name, user_conditions, opts)
             return nil
         end
 
-        local obj, err = iter:get()
+        local tuple, err = iter:get()
         if err ~= nil then
             error(string.format("Failed to get next object: %s", err))
         end
 
+        local obj, err = utils.unflatten(tuple, iter.space_format)
```

Comment:
I suggest adding a flag to opts for forcing unflatten and return here tuples by default as in other methods. Or it will cause additional code for flattening in custom proxy methods

Reply:
`use_tomap` option is introduced (https://github.com/tarantool/crud/pull/39/commits/4a1b3644d5f4c39f5a5133a002860b1cda631dca)

Not sure we should implement it just now.

Reply:
Maybe it would be better to implement this option in separated patch.

Reply:
It costs nothing since it is already implemented, and it blocks our connector development. Since that code has been already accepted, can we port it here?

Reply:
Moved to separate PR - https://github.com/tarantool/crud/pull/43